ftw/dashboard/dragndrop/browser/resources/dnd_dashboard.js

```diff
@@ -90,12 +92,20 @@ jQuery(function($){
       type :      'POST',
       url :       './ftw.dashboard.dragndrop-update_order',
       data :      customSerialization(),
+      beforeSend: function (request)
+      {
```

Comment:
I'd move the `{` to the end of the line above to preserve local consistency. Same at the other places..